### ISSUE TITLE
multi-field dates need group labels

### ISSUE THREAD

**Issue by pdpinch** [mitodl/micromasters#1683]
for example, date of birth in the profile form

>     Use fieldset or role="group" and aria-labelledby to label the group
>     use label element to label each individual field (month, day, year)

**Comment by pdpinch** [comment]
@singingwolfboy did this get fixed #1718

**Comment by singingwolfboy** [comment]
Yes, although this issue only applies to dates. The gender radio buttons are still not accessible, for example.

**Comment by pdpinch** [comment]
I opened a new issue for the gender radio buttons
